bots/run-queue

```diff
@@ -44,13 +44,15 @@ def consume_webhook_queue(channel, amqp):
     event = body['event']
     request = body['request']
     repo = None
+    cmd = ':' # nop
```

Comment:
If that ever survives, it will fail trying to add `+= ['--repo', repo]` to it. If it handles the "no command" special case somewhere, then let's please use `None`.

Reply:
Done!